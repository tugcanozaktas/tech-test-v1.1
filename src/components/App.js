import '../styles/App.css';

function App() {
  return (
    <div className="App">
      <img src='https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg' alt='nasa-logo' className='nasa-logo' />
    </div>
  );
}

export default App;
