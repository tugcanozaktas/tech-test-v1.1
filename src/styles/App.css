.App{
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 100%;
}

.nasa-logo{
  height: 400px;
  width: 800px;
}