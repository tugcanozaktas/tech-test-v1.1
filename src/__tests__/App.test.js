import React from 'react';
import { render } from '@testing-library/react';
import App from '../components/App.js';

describe("App", () => {
    const { asFragment } = render(<App />);

    it("renders correctly", () => {
        expect(asFragment()).toMatchSnapshot();
    })
})
