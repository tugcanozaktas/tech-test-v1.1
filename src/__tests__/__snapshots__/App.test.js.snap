// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      alt="nasa-logo"
      class="nasa-logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <div
      class="search"
    >
      <form
        class="search-form"
      >
        <input
          class="search-input"
          type="text"
        />
        <button
          class="search-button"
          type="submit"
        >
          Search!
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
